.github/workflows/tests.yml

```diff
@@ -28,6 +28,12 @@ jobs:
             os: macos-12
           - node: 19
             os: windows-2022
+          - node: 20
```

Comment:
Now that we're adding 20, perhaps we can drop 14 and expand 16 to also include macos and windows?
I also question whether testing on 19 is necessary. I've always considered odd numbered node versions to be unstable.

Reply:
Yes, updating runners in https://github.com/google/wireit/pull/729

I'd like to get one release out that supports 14-20 and then an immediate follow up that drops that to 16-20 (followed shortly by another that drops it to 18-20)